v1/pdf.yaml

```diff
@@ -0,0 +1,84 @@
+swagger: '2.0'
+info:
+  version: '1.0.0-beta'
+  title: MediaWiki PDF API
+  description: Page PDF Render API
+  termsOfService: https://github.com/wikimedia/restbase#restbase
+  contact:
+    name: Services
+    email: services@lists.wikimedia.org
+    url: https://www.mediawiki.org/wiki/Services
+  license:
+    name: Apache licence, v2
+    url: https://www.apache.org/licenses/LICENSE-2.0
+paths:
+  /pdf/{title}:
+    x-route-filters:
+      - path: ./lib/revision_table_access_check_filter.js
+        options:
+          redirect_cache_control: '{{options.response_cache_control}}'
+    get:
+      tags:
+        - Page content
+      summary: Get a page as PDF
+      description: |
+        Renders the page content as PDF.
+
+        Stability: [experimental](https://www.mediawiki.org/wiki/API_versioning#Experimental)
+      produces:
+        - application/pdf
+      parameters:
+        - name: title
+          in: path
+          description: "Page title. Use underscores instead of spaces. Example: `Main_Page`."
+          type: string
+          required: true
+      responses:
+        '200':
+          description: The PDF render of an article
+          schema:
+            type: file
+          # TODO: Add an etag somehow. How???
```

Comment:
I think that doing another round trip to Cassandra is not such a big penalty, given that we don't expect much traffic here and the amount of time it takes to actually render a page in pdf.